test/spherical_voxel_grid_test.pcd

```diff
@@ -0,0 +1,27 @@
+# .PCD v0.7 - Point Cloud Data file format
+VERSION 0.7
+FIELDS x y z intensity
```

Comment:
If all intensity values are gonna be zero, there's no point in having that field.